plugins/cpp/parser/src/cppparser.cpp

```diff
@@ -329,19 +332,142 @@ int CppParser::worker(const clang::tooling::CompileCommand& command_)
 
 CppParser::CppParser(ParserContext& ctx_) : AbstractParser(ctx_)
 {
-  (util::OdbTransaction(_ctx.db))([&, this] {
-    for (const model::BuildAction& ba : _ctx.db->query<model::BuildAction>())
-      _parsedCommandHashes.insert(util::fnvHash(ba.command));
-  });
 }
-  
+
 std::vector<std::string> CppParser::getDependentParsers() const
 {
   return std::vector<std::string>{};
 }
 
+bool CppParser::preparse()
+{
+  std::vector<model::FilePtr> filePtrs(_ctx.fileStatus.size());
+
+  std::transform(_ctx.fileStatus.begin(),
+                 _ctx.fileStatus.end(),
+                 filePtrs.begin(),
+                 [this](const auto& item)
+                 {
+                   if (item.second == IncrementalStatus::MODIFIED)
+                   {
```

Comment:
Please remove the brackets from these one-liner branches.

Reply:
The [coding convention](https://github.com/Ericsson/CodeCompass/blob/master/doc/coding_conventions.md) states nothing on this matter and the existing codebase is also very inconsistent on the usage of brackets in such cases.

Since no uniform coding style is applied regarding this, a personal coding style preference should not be reflected as change requests.

Reply:
Yeah, because obviously in a project where code reviews hang for **multiple years** and where ***zero** developers exist officially* a code style change could go in any time, right?

Reply:
We are not a mind-readers @whisperity , we cannot comply with coding conventions which are not mentioned anywhere and cannot be deduced from the codebase, because it is inconsistent.

If you think that the current coding convention is not broad or strict enough, please suggest its extension (we have weekly meetings!), but requesting changes on a developed feature based on your **personal** preferences is both unprofessional and unacceptable.